pom.xml

```diff
@@ -17,7 +17,7 @@
   <maven.compiler.source>${java.version}</maven.compiler.source>
 
   <imageBuilder>/usr/bin/podman</imageBuilder>
-  <baseImage>registry.access.redhat.com/ubi8/openjdk-11</baseImage>
+  <baseImage>registry.access.redhat.com/ubi8/openjdk-11:1.3-3.1593114401</baseImage>
```

Comment:
Rather than appending it directly to the `baseImage` property, what do you think about creating a new property like `baseImageTag` and specifying the version there? Further down in the file, when the `baseImage` is referenced by the Jib plugin, you would then specify the `from` image like `<from>${baseImage}:${baseImageTag}</from>`.

Reply:
ah, looks better, done!